python/ray/serve/pipeline/deployment_function_node.py

```diff
@@ -92,48 +89,8 @@ def _copy_impl(
             other_args_to_resolve=new_other_args_to_resolve,
         )
 
-    def _execute_impl(self, *args, **kwargs) -> ObjectRef:
-        """Executor of DeploymentFunctionNode by calling .remote() on the
-        deployment handle.
-
-        Deployment method always default to __call__.
-        """
-        return self._deployment_handle.remote(*self._bound_args, **self._bound_kwargs)
-
     def __str__(self) -> str:
         return get_dag_node_str(self, str(self._body))
 
     def get_deployment_name(self):
         return self._deployment_name
-
-    def get_import_path(self):
```

Comment:
@shrekris-anyscale FYI about removing import path here and other deployment nodes